Maquetado/home.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>Lista de Libros</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body class="background">
```

Comment:
Use a div and apply the class in the div instead of the body. The body not must be styled.